### PR TITLE
Fixes after merging "generateRandom" table function

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Addition to #8994

### CODE REVIEW

File: dbms/src/Storages/StorageGenerateRandom.cpp [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+            ColumnWithTypeAndName argument{std::move(size_column), std::make_shared<DataTypeUInt32>(), "size"};
+
+            Block block
+            {
+                argument,
+                {nullptr, type, "result"}
+            };
+
+            FunctionFactory::instance().get("randomPrintableASCII", context)->build({argument})->execute(block, {0}, 1, limit);
```
Comment (qoega) [inline dbms/src/Storages/StorageGenerateRandom.cpp]: We can't use randomPrintableASCII as it has no way do have deterministic result. There is no way to set random_seed.

File: dbms/src/Storages/StorageGenerate.cpp [hunk 2]
```diff
@@ -1,491 +0,0 @@
-#include <Storages/IStorage.h>
```
Comment (qoega) [inline dbms/src/Storages/StorageGenerate.cpp]: Lets try to preserve history for a file. Probably mv file in a separate commit can help.

**Review by qoega — CHANGES_REQUESTED** [review]

**Comment by qoega** [comment]
There is a better name suggestion: `datagen`

File: dbms/src/Storages/StorageGenerateRandom.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply dbms/src/Storages/StorageGenerateRandom.cpp]: Ok.

File: dbms/src/Storages/StorageGenerate.cpp (hunk 2 above)
Comment (alexey-milovidov) [reply dbms/src/Storages/StorageGenerate.cpp]: I already made `git mv` but then the history is lost.

**Comment by alexey-milovidov** [comment]
Normalize performance tests: in next PR.

**Pull request merged**